forms/admin.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.admin import ModelAdmin
+from django.contrib.contenttypes.models import ContentType
+from django import forms
+from django.forms import ModelForm
+
+from formula_one.models import (
+    ContactInformation,
+    LocationInformation,
+    SocialInformation,
+)
+
+admin_form_dict = {
+    'ContactInformation': ContactInformation,
+    'LocationInformation': LocationInformation,
+    'SocialInformation': SocialInformation,
+}
+
+
+def return_admin_form(class_name):
+    """
+    Return the model admin form
+    :param class_name: class name of the model
+    :return: admin form class
+    """
+
+    class Form(ModelForm):
+        """
+        ModelForm corresponding to the model
+        """
+
+        entity_content_object = forms.CharField(widget=forms.Select)
+
+        def __init__(self, *args, **kwargs):
```

Comment:
If you're just calling `super()`, don't define the function at all!

Reply:
Oh sorry :sweat_smile: This function was used before. Forgot to delete it.

Reply:
There are a couple like this.

Reply:
@dhruvkb what do you mean?